src/shared/components/boardCard/boardCard.js

```diff
@@ -24,7 +28,8 @@ BoardCard.propTypes = {
   name: PropTypes.string.isRequired,
   src: PropTypes.string.isRequired,
   alt: PropTypes.string.isRequired,
-  role: PropTypes.string.isRequired
+  role: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired
```

Comment:
Instead of conditional rendering, you can make this a non-required PropType, and then write this:

```
BoardCard.defaultProps = {
  description: null
};
```

Now your conditional rendering can look like this:
```
{description !== null && <p>{description}</p>}
```